packages/eslint-plugin-office-addins/src/rules/no-context-sync-in-loop.ts

```diff
@@ -0,0 +1,30 @@
+import { TSESTree } from "@typescript-eslint/typescript-estree";
+
+export = {
+  name: "no-context-sync-in-loop",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      loopedSync: "Calling context.sync() inside a loop can lead to bad performance",
+    },
+    docs: {
```

Comment:
This docs section is just for metadata? Or the client does see some of that?

Reply:
I believe it's original design for for eslint documentation generation, but at least some of this (the url specifically) is getting used by VS Code.  Still have to figure out the rest.